src/server/renderer/render_canvas.cpp

```diff
@@ -31,7 +31,7 @@
 
 #include "config.hpp"
 
-#include <cairo/cairo-svg.h>
+#include <cairo-svg.h>
```

Comment:
Our cmake seems to find path directly to cairo includes. Let's see what Travis CI says.